src/flow/index.ts

```diff
@@ -0,0 +1,256 @@
+type UnknownFn = (...params: unknown[]) => unknown
+type AnyFn = (...params: any[]) => any
+
+/**
+ * Lets you perform a flow of function operations sequentially
+ *
+ * @param functions - The function operations that you want to perform on a given value
+ * @returns A function that accepts an initial value to perform the above functions on
+ *
+ * @public
+ */
+export default function flow<First extends unknown, Output>(
```

Comment:
I'm wondering why does it extend `unknown`?

Reply:
`unknown` so that the user can explicity add types after overloading 10 functions. using `any` would allow any operation and that could break on runtime.

Reply:
Should I change it to `any`?

Reply:
No, seems like `unknown` is needed. However, I would like to see a type test for it using `expectType` or `// @ts-expect-error`